src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import classNames from "classnames";
+
+import { WithPoster } from "types/posters";
+import { PosterVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import "./PosterPreview.scss";
+
+export interface PosterProps {
+  posterVenue: WithId<WithPoster<PosterVenue>>;
+  onClick: () => void;
+}
+
+export const PosterPreview: React.FC<PosterProps> = ({
+  posterVenue,
+  onClick,
+}) => {
+  // TODO: Remove poster object and put all of its properties into the venue itself
+  const { title, pdfUrl, author, categories } = posterVenue.poster;
+
+  const posterClassnames = classNames("poster-preview", {
+    "poster-preview--live": posterVenue.isLive,
+  });
+
+  return (
+    <div className={posterClassnames} onClick={onClick}>
+      <div className="poster-preview__pdf">
+        <iframe
+          src={pdfUrl}
+          width="100%"
+          title={title}
+          className="poster-preview__pdf-iframe"
+        />
+      </div>
+
+      <p className="poster-preview__title">{title}</p>
+      {categories?.length > 0 && (
+        <div className="poster-preview__categories">
+          {categories.map((category) => (
+            <div
+              key={category.title + category.color}
+              className="poster-preview__category"
+              style={{
+                backgroundColor: category.color,
+              }}
+            >
+              {category.title}
+            </div>
+          ))}
+        </div>
+      )}
```

Comment:
FIxed in https://github.com/sparkletown/sparkle/pull/1299/commits/f4c868471dcad8c0b597c686ef3a796ef5c4aa89

Yes, `category.title` should be sufficient alone. Keep in mind that this is within one poster preview. It would mean that the data is corrupted, if the keys would not be unique

Reply:
> Keep in mind that this is within one poster preview. It would mean that the data is corrupted, if the keys would not be unique

Given I don't believe there is any definition of the data/where its coming from/it's shape/format/filtering/etc in this PR, that's not an assumption that I could reasonably make.